src/main/resources/application.properties

```diff
@@ -7,3 +7,7 @@ logging.level.org.mongodb.driver.connection=error
 logging.level.org.springframework.data.convert=off
 spring.config.location=file:./src/main/resources/application.properties
 spring.data.mongodb.uri=
+
+bdk.host=acme.symphony.com
```

Comment:
we need to migrate the config.json here (and drop the config.json file)

Reply:
and we also need to figure out with @ystan- how it is configured when deployed for real

Reply:
I'm not sure about the RSA key path. @ystan- Could you clarify the config of the poll bot please?